monkeys/src/main/kotlin/com/wire/kalium/monkeys/importer/TestData.kt

```diff
@@ -54,42 +54,67 @@ data class GroupConfig(
 )
 
 @Serializable
-data class Action(
+data class ActionConfig(
     @SerialName("description") val description: String,
-    @SerialName("type") val type: ActionType,
+    @SerialName("config") val type: ActionType,
     @SerialName("count") val count: UserCount,
-    @SerialName("repeatDuration") val repeatDuration: UInt
+    @SerialName("repeatDuration") val repeatDuration: ULong = 0u
```

Comment:
What does `repeatDuration` mean? 
Is it the interval between repetitions? Maybe `repeatInterval`?

Reply:
yes. I'll rename it